### PR TITLE
K8s: revise openshift cli getting started 

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-870
Staging site: https://docs.redis.com/staging/jira-doc-870/kubernetes/deployment/openshift/openshift-cli/

### CODE REVIEW

**Comment by kaitlynmichael** [comment]
Staging site: https://docs.redis.com/staging/jira-doc-870/kubernetes/deployment/openshift/openshift-cli/

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 1]
```diff
@@ -22,317 +22,229 @@ aliases: [
...
+    | Redis-enterprise-operator-x-x    | 1/1   | Running | 0        | 5m  |
+    ```
 
-    - name: “your_cluster_name” - For example “demo-cluster”
-    - nodes: number_of_nodes_in_the_cluster - Must be an [uneven number of 3 or more](https://redis.com/redis-enterprise/technology/highly-available-redis/)
-    - uiServiceType: service_type - Service type value can be either `ClusterIP` or `LoadBalancer`. This is an optional configuration based on [k8s service types](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/). The default is `ClusterIP`.
+## Configure the admission controller
 
-    - storageClassName: “gp2“ - This specifies the [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) used for the persistent disks in your nodes. For example, AWS uses “gp2” as a default, GKE uses “standard”, and Azure uses "default".
+1. Check that the secret has been created.
```
Comment (lanceleonard) [inline content/kubernetes/deployment/openshift/openshift-cli.md:122]:
```suggestion
1. Verify the secret has been created.
```

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 2]
```diff
@@ -22,317 +22,229 @@ aliases: [
...
 
-    The command returns a confirmation response such as:
+    ```
 
-    ```sh
-    NAME AGE
+1. Apply the patch.
 
-    Your_cluster_name 17s
+    ``bash
```
Comment (lanceleonard) [inline content/kubernetes/deployment/openshift/openshift-cli.md]:
```suggestion
    ```bash
```

**Review by lanceleonard — CHANGES_REQUESTED** [review]
LGTM, though a couple of things need to be tweaked.

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 3]
```diff
@@ -22,317 +22,230 @@ aliases: [
...
+      spec:
+       evictionPolicy: illegal
+      EOF
+      ```
 
-1. Apply port forwarding to your Cluster:
 
-    ```sh
-    oc port-forward your_cluster_name-0 8443:8443
+    You should see an error that the admission controller webhook `redb.admission.redislabs`.
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/openshift-cli.md:213]: Looks like this sentence is missing something.

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 4]
```diff
@@ -22,317 +22,230 @@ aliases: [
...
     ```
 
-    {{< note >}}
-- your_cluster_name-0 is one of your cluster pods. Consider running the port-forward command in the background.
-- The Openshift UI provides tools for creating additional routing options, including external routes. These are covered in [RedHat Openshift documentation](https://docs.openshift.com/container-platform/4.8/networking/ingress-operator.html).
-    {{< /note >}}
+## Create a Redis Enterprise database (REDB) custom resource
 
-    Next, create your database.
+The operator uses the instructions in the REDB custom resources to manage databases on the Redis enterprise cluster.
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/openshift-cli.md:221]:
```suggestion
The operator uses the instructions in the REDB custom resources to manage databases on the Redis Enterprise cluster.
```

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 5]
```diff
@@ -22,317 +22,230 @@ aliases: [
...
 
-1. Open a browser window and navigate to the Redis Enterprise admin console at: `localhost:8443`
+1. Create a `RedisEnterpriseDatabase` custom resource.
 
-    ![getting-started-kubernetes-openshift-image5]( /images/rs/getting-started-kubernetes-openshift-image5.png )
+    We've provided an example below that will create a database for testing purposes. For production databases, see documentation for [creating a database]({{<relref "/kubernetes/re-databases/db-controller.md#create-a-database" >}}) and [database options]({[<relref "/kubernetes/reference/db-options.md">]}).
```
Comment (rrelledge) [inline content/kubernetes/deployment/openshift/openshift-cli.md:225]:
```suggestion
    We've provided an example below that will create a database for testing purposes. For production databases, see documentation for [creating a database]({{<relref "/kubernetes/re-databases/db-controller.md#create-a-database" >}}) and [database options]({{<relref "/kubernetes/reference/db-options.md">}}).
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I left a couple of small suggestions.

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 6]
```diff
@@ -22,317 +22,230 @@ aliases: [
...
+      spec:
+       evictionPolicy: illegal
+      EOF
+      ```
 
-1. Apply port forwarding to your Cluster:
 
-    ```sh
-    oc port-forward your_cluster_name-0 8443:8443
+    You should see an error that the admission controller webhook `redb.admission.redislabs`.
```
Comment (kaitlynmichael) [inline content/kubernetes/deployment/openshift/openshift-cli.md:213]:
```suggestion
    You should see an error from the admission controller webhook `redb.admission.redislabs`.
```

File: content/kubernetes/deployment/openshift/openshift-cli.md (hunk 3 above)
Comment (kaitlynmichael) [reply content/kubernetes/deployment/openshift/openshift-cli.md:213]: I meant "from" not "that". Good catch!

File: content/kubernetes/deployment/openshift/openshift-cli.md [hunk 7]
```diff
@@ -22,317 +22,230 @@ aliases: [
...
+## Create a Redis Enterprise database (REDB) custom resource
 
-    Next, create your database.
+The operator uses the instructions in the REDB custom resources to manage databases on the Redis Enterprise cluster.
 
-1. Open a browser window and navigate to the Redis Enterprise admin console at: `localhost:8443`
+1. Create a `RedisEnterpriseDatabase` custom resource.
 
-    ![getting-started-kubernetes-openshift-image5]( /images/rs/getting-started-kubernetes-openshift-image5.png )
+    We've provided an example below that will create a database for testing purposes. For production databases, see documentation for [creating a database]({{<relref "/kubernetes/re-databases/db-controller.md#create-a-database" >}}) and [database options]({{<relref "/kubernetes/reference/db-options.md">}}).
```
Comment (kaitlynmichael) [inline content/kubernetes/deployment/openshift/openshift-cli.md:225]:
```suggestion
    The following example creates a database for testing purposes. For production databases, see documentation for [creating a database]({{<relref "/kubernetes/re-databases/db-controller.md#create-a-database" >}}) and [database options]({{<relref "/kubernetes/reference/db-options.md">}}).
```

Comment (kaitlynmichael) [inline content/kubernetes/deployment/openshift/openshift-cli.md:225]:
```suggestion
    We've provided an example below that will create a database for testing purposes. For production databases, see [creating a database]({{<relref "/kubernetes/re-databases/db-controller.md#create-a-database" >}}) and [database options]({{<relref "/kubernetes/reference/db-options.md">}}).
```

**Review by lanceleonard — APPROVED** [review]
LGTM

**Pull request merged**